modules/material/locales/es/LC_MESSAGES/material.po

```diff
@@ -69,10 +69,10 @@ msgid "{footer:copyright}"
 msgstr "El uso no autorizado de este sitio está prohibido y puede ser sometido a procesamiento civil y penal."
 
 msgid "{login:title}"
-msgstr "Iniciar sesión en %idpName%"
+msgstr "%idpName% Iniciar sesión"
```

Comment:
Is this correct grammar for spanish? I just wonder if %idpName% would be an adjective. If so, then it should probably be after the noun (Iniciar sesión). And French is probably the same.